chainer/links/loss/crf1d.py

```diff
@@ -19,10 +20,14 @@ class CRF1d(link.Link):
         cost (~chainer.Variable): Transition cost parameter.
     """
 
-    def __init__(self, n_label):
+    def __init__(self, n_label, initialW=None):
```

Comment:
Do you think `initialW` is preferred over `initial_cost`, as `cost` is the name of the parameter for which it'll be used?

Reply:
I'm not sure what is best, you think initial_cost is better, right?
I use `initialW` only to keep the consistency with other modules.

Reply:
Yes exactly. The convention should probably be to associate it with the name of the parameter. Could you update is you think?